src/terms.js

```diff
@@ -133,6 +133,7 @@ export default [
     'front lines',
     'functional',
     'full benefit',
+    'future proof',
```

Comment:
```suggestion
    'future.proof',
```

To also cover "future-proof"